### ISSUE TITLE
OSError: Permission Denied in ./runserver.sh on vagrant

### ISSUE THREAD

**Issue by sbenthall** [roundware/roundware-server#422]
After correcting the requirements as per #421, I've continued with the vagrant installation instructions.

I'm getting the following error about permissions installing `wcwidth`:

https://gist.github.com/sbenthall/a21990cd57a1e83fbb5650425d7617fe

I'm a bit confused about the error because I'm assuming the dependencies are being installed on a virtual machine, and that the `roundware` user would have permissions to do the install....

**Comment by sbenthall** [comment]
Indeed, it is running on the virtual machine. `sudo ./runserver.sh` does not do the trick:

https://gist.github.com/sbenthall/0cd74dbc19503d1675120a7c2bfea3ba

**Comment by hburgund** [comment]
I'm not sure the exact steps you are taking, but I am not experiencing any issues getting the server up and running locally using `vagrant` with the following steps:

1. grab fresh `admin-updates` branch
2. update to `django-sortedm2m==3.1.1`
3. `vagrant up`
4. `vagrant ssh`
5. `cd roundware-server/scripts`
6. `./runserver.sh` (no `sudo`)

I haven't tested the server extensively, but it all installs and runs without any errors for me. LMK if your process deviates from this in any way or if you are experiencing errors using the same process.

**Comment by sbenthall** [comment]
>     1. grab fresh `admin-updates` branch

Aha; this is how my process deviates. I got this error when using the `develop` branch.

You mentioned `admin-updates` in personal communication but I forgot to switch over.

I'll try again with `admin-updates` but wanted to flag that I can't find any mention of using this alternative branch in the documentation, and so it's quite confusing. It looks like PR #420 is the merge PR, but it had some failing tests?

I wouldn't know which branch to submit PRs with fixes (such as the dependency update from #421 ) to.

**Comment by sbenthall** [comment]
Confirming that with the `admin-updates` branch, the `./runserver.sh` script executes!
